test/binCases/env/string/test.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+module.exports = function testAssertions(code, stdout, stderr) {
+	code.should.be.exactly(0);
+
+	stdout.should.be.ok();
+	stdout[0].should.containEql("Hash: ");
+	stdout[1].should.containEql("Version: ");
+	stdout[2].should.containEql("Time: ");
+	stdout[3].should.containEql("Environment (--env): ");
```

Comment:
`Environment (--env): foo`?

Reply:
Updated to include the expected out put of `Environment (--env): \"foo\""`.